### PR TITLE
#357 - Add space below lists on website 

### PR DESCRIPTION
**Pull request by shatakshi-shukla**
Add space below lists on website - #357.

### CODE REVIEW

File: ideas.html [hunk 1]
```diff
@@ -520,12 +520,14 @@ <h1 class="large-h1">Subscribe to get announcements for upcoming Coding Stipends
 	font-weight:bold;
 }
 
+
```
Comment (realslimshanky) [inline ideas.html]: please remove this unnecessary line.

**Comment by shatakshi-shukla** [comment]
@realslimshanky the required changes have been made. And the preview URL is  - https://shatakshi-shukla.github.io/labs.fossasia.org/ideas.html .
screenshot 2019-02-17 at 9 54 33 pm

**Comment by shatakshi-shukla** [comment]
CNAME file has been deleted.

**Comment by mariobehling** [comment]
Don't do anything to the CNAME. Check your PR.

**Comment by shatakshi-shukla** [comment]
The changes to the CNAME have been reverted.

**Pull request merged**